recipes/snntorch/meta.yaml

```diff
@@ -1,14 +1,14 @@
-{% set name = "aiosignal" %}
```

Comment:
Please don't modify/delete existing recipes in the repository, only add new files.

Reply:
Thanks for the feedback. I've reset the branch + tried again by only adding new files in the latest merge.